docs/stylesheets/style.css

```diff
@@ -632,11 +632,13 @@ p {
 @media only screen and (max-width: 768px) {
   .nav__items {
     flex-direction: column;
+    display: flex;
     position: fixed;
-    top: calc(calc(var(--gap-lg) * 2 + 32px) + 0.5rem);
+    top: 0;
     left: 0;
-    width: 100vw;
-    background-color: var(--color-navigation);
+    width: 100%;
```

Comment:
in this context there isn't a difference in 100vw (100 % of viewport width) vs 100%, so it's okay to keep it as before

Reply:
With the viewport still there, I'm seeing this width in my mobile simulator (https://www.webmobilefirst.com/en/). 
<img width="442" alt="Screen Shot 2023-11-08 at 11 38 49 AM" src="https://github.com/truera/trulens/assets/60949774/05bf1e80-c792-4816-92b4-a75cd495859d">

Reply:
oh interesting - then yes let's keep it